public/index.html

```diff
@@ -8,6 +8,7 @@
     <link rel="stylesheet" href="/css/bootstrap.min.css">
     <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet" integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous">
     <link rel="stylesheet" type="text/css" href="./css/application.css">
+    <script type="text/javascript" src="https://maps.googleapis.com/maps/api/js?libraries=visualization&key=__API_KEY_HERE__"></script>
```

Comment:
we'll leave it in. client side google api keys can't be hidden, but that's ok. we just need to generate a new one later that specifies which hosts can make requests with it.